src/gui/Src/Tracer/TraceBrowser.cpp

```diff
@@ -1373,3 +1395,8 @@ void TraceBrowser::updateSlot()
         reloadData();
     }
 }
+
+void TraceBrowser::toggleAutoDisassemblyFollowSelection()

```

Comment:
Probably it's better to rename this to `toggleAutoDisassemblyFollowSelectionSlot` to follow the convention of the other slots...

Reply:
Alright, I renamed it